README.md

```diff
@@ -99,11 +99,14 @@ var app = express()
 var whitelist = ['http://example1.com', 'http://example2.com']
 var corsOptions = {
   origin: function (origin, callback) {
-    if (whitelist.indexOf(origin) !== -1) {
-      callback(null, true)
-    } else {
-      callback(new Error('Not allowed by CORS'))
-    }
+    // Pass true to the callback to echo back the incoming origin
```

Comment:
Can this be moved to the documentation text about the function?

Reply:
Done 👍 I've added a bit more on what passing up an error will do as well, vs saying not to do it and not explaining why.